arm-multilib/multilib-generate.py

```diff
@@ -238,7 +238,8 @@ def get_extension_list(clang, triple):
 def generate_extensions(args):
     aarch64_features = get_extension_list(args.clang, "aarch64-none-eabi")
     aarch32_features = get_extension_list(args.clang, "arm-none-eabi")
-    all_features = set(aarch64_features) | set(aarch32_features)
+    all_features = list(aarch64_features)
+    all_features.extend(feat for feat in list(aarch32_features) if feat not in all_features)
```

Comment:
Just determinism: I was trying to check the generated `multilib.yaml` file was not any different after my changes in this patch, but discovered the output of this script is different every time it is run because of the casting to sets. Otherwise order of the features doesn't seem to be important; I might have thought it neater to have aarch64 features grouped together though a sort would be easier.